content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported. All JVM-based languages, i.e. Scala, Groovy, Kotlin, Clojure, etc. are supported.
+
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][2].
```

Comment:
Github's being weird, but these comments are still open:

Tracing is on by default so we don't need to tell them to enable tracing.

Can we link straight to getting started rather than requiring an extra hop through a footnote?

And one more- it's a little awkward to say "To enable tracing" since they might be here only for profiling. Like of course they _do_ need to enable tracing but maybe we could phrase it more in terms of what they're trying to do. Actually, I now pretty strongly believe that we shouldn't link them to that page at all. They need _1_ line from it (`wget -O dd-java-agent.jar 'https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.datadoghq&a=dd-java-agent&v=LATEST'`), and 99% of what's there is irrelevant to trying profiling and will kill the conversion rate for anyone specifically looking to try profiling. For example, literally the first thing on the page tell them to jump through the hoops of installing the Datadog Agent, which isn't needed for profiling.